src/task2/task2.services.ts

```diff
@@ -0,0 +1,50 @@
+import { Injectable , NotFoundException} from "@nestjs/common";
+import { Flight } from "./task2.model";
+
+@Injectable()
+
+export class FlightService{
+    flight: Flight[] = [];
+
+    bookFlight(source: string, destination: string, date: string, fare: number, seats: number, flightno: string,seatno:number){
+        const id = Math.random().toString();
+        const newFlight = new Flight(id, fare,source, destination, seatno, seats, flightno , date);
+        if(seatno > seats){

```

Comment:
how client can send you total number of seats ? it has be stored at server side , isn't it?